tests/EvmAcceptanceTests/test/Transfer.ts

```diff
@@ -1,6 +1,7 @@
-import {expect} from "chai";
-import {parallelizer} from "../helpers";
-import {ethers} from "hardhat";
+import { expect } from "chai";
```

Comment:
Please run lint on the code. {} spaces are not consistent throughout the code.

Reply:
Will do thanks.